### DISCUSSION TITLE
[Feature] Conditional Partner Share

### DISCUSSION THREAD

**Discussion by dadino** [immich-app/immich#14392] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I'd like to enable Partner Share with my partner, but we constantly take picture of stuff that the other one doesn't need. Partner Share, just like in Google Photos, is too black and white of a choice.

What I'd love to do is filter what I share with my partner. When enabling Partner Share in the UI, a new Filter check box would appear, when enabled, show additional UI to create OR conditions for the filter (like the UI to create path in external libraries).

Each condition could have multiple parameters, like "photos with both Alice and Bob in it" or "photos/videos in France, taken from 2023 onward" or "photos with tags: KID, GAMES".

We have a newborn (let's call him Bob), so in my case I'd share with my partner (let's call her Alice) with these filters:
photos/videos with Alice
photos/videos with Bob
photos/videos with Alice's parents
photos/videos with tag "for Alice"

I'd also add our parents (Bob grandparents) to their own Partner Share implementation, with their custom filters. (my mother would see photo with "My mother", "My father + Bob", "media with tag "for grandparents""

Looking at the roadmap, this could be rolled in the workflow feature, if that feature allows to add media to Partner Share, using the aforementioned filters.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by danieldietzler** [comment]
Dupe of #12614
